### PR TITLE
Update Main Repository Readme.md and GroundwaterDupuitPercolator readme.md

### PR DESCRIPTION
**Pull request by kbarnhart**
Along with #1110 this PR addresses #1108 (associated with [the JOSS review](https://github.com/openjournals/joss-reviews/issues/1935) for this component.

- [ ] This Readme will need to be updated before it can be merged after #1108 has been finalized because there is a link to a new page about dependencies in the docs.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -96,8 +96,6 @@ conventions that we would like you to follow.
 
 * Create a new folder under `landlab/components` that will hold your
   new component.
-* Be aware that your component will have to be able to work with both
-  Python 2 and Python 3.
 
 ### Landlab 2 is Python >3.6 only.
```
Comment (nathanlyons) [inline CONTRIBUTING.md]: ">=3.6" or "3.6 and later versions"? Travis tests include 3.6.

File: README.rst [hunk 2]
```diff
@@ -20,33 +20,238 @@
...
+  visualization.
 
-This open-source manuscript is a gateway for entering the Landlab world:
+In addition Landlab contains a set of Jupyter notebook tutorials providing
+introduction to core concepts and examples of use.
 
-https://www.earth-surf-dynam.net/5/21/2017/
+Landlab was desiged for disciplines that quantify Earth surface dynamics such
+as geomorphology, hydrology, glaciology, and stratigraphy, it can also be used
+in related fields. Scientists who want to use a this type of model often build
```
Comment (nathanlyons) [inline README.rst]: extra word `use a this`

File: README.rst [hunk 3]
```diff
@@ -20,33 +20,238 @@
...
+
+Watch the webinar `Landlab Toolkit Overview <https://csdms.colorado.edu/wiki/Presenters-0407>`_
+ at CSDMS to learn more.
+
+How do I install Landlab?
+-------------------------
+
+First you'll need a Python distribution and either the conda or pip package
+manager. If you don't know what you want, we recommend the Anaconda Python
+distribtuion.
```
Comment (nathanlyons) [inline README.rst]: typo `distribtuion`

File: README.rst [hunk 4]
```diff
@@ -20,33 +20,238 @@
...
+Landlab uses pytest to discover and run tests. Presuming you have used a source
+code installation with the above conda environment, you will be able to test
+your install with
+
+.. code-block::
+
+    $ pytest
+
+from within the ``landlab_dev`` conda environment. Additional instructions
+can be found `here <>`_.
```
Comment (nathanlyons) [inline README.rst]: link forthcoming? maybe <https://landlab.readthedocs.io/en/latest/development/practices/writing_tests.html>

File: README.rst [hunk 5]
```diff
@@ -20,33 +20,238 @@
...
+
+What are Landlab's dependencies?
+--------------------------------
+
+The core package dependencies are specified by ``requirements.txt`` and used
+by ``setup.py``. There are some additional dependencies that exist for
+running the notebooks or modifying the source code and testing.
+
+Details of how we structure our dependencies can be found `here <>`_.
```
Comment (nathanlyons) [inline README.rst]: link forthcoming? maybe <https://landlab.readthedocs.io/en/latest/development/practices/dependencies.html#how-landlab-specifies-dependencies>

File: README.rst [hunk 6]
```diff
@@ -20,33 +20,238 @@
...
+------------------------------------------------------
+
+The Landlab package contains a directory called ``notebook`` which contains
+Jupyter notebooks describe core concepts and give examples of using components.
+The file ``landlab/notebooks/welcome.ipynb`` provides a table of contents to
+the notebooks and is the recommended starting place. To launch an instance of
+Binder and explore the notebooks click
+`here <https://mybinder.org/v2/gh/landlab/landlab/release?filepath=notebooks/welcome.ipynb>_`.
+
+In addition there are a set of curated teaching notebooks located in
```
Comment (nathanlyons) [inline README.rst]: "In addition there are a set of notebooks curated to teach physical processes located in"

File: README.rst [hunk 7]
```diff
@@ -20,33 +20,238 @@
...
+------------------------------------------------------------------
+
+Thank you for your interest! Refer to ``CONTRIBUTING.md`` and
+`this <https://landlab.readthedocs.io/en/master/development/index.html#development>`_
+page in the documentation that describes contribution guidelines.
+
+How is the Landlab package structured?
+--------------------------------------
+
+The `following page <>` in the documentation describes the package structure.
```
Comment (nathanlyons) [inline README.rst]: not sure of the link

File: README.rst [hunk 8]
```diff
@@ -20,33 +20,238 @@
...
+ - Jordan Adams (Tulane)
+ - Eric Hutton (CU)
+ - Jenny Knuth (CU)
+ - Katy Barnhart (CU)
+ - Margaux Mouchene (CU)
+ - Christina Bandaragoda (UW)
+ - Nathan Lyons (Tulane)
 
-https://github.com/landlab/landlab
+We welcome all community contributions.
```
Comment (nathanlyons) [inline README.rst]: We welcome all community contributions. Please see https://github.com/landlab/landlab/blob/master/CONTRIBUTING.md.

File: README.rst [hunk 9]
```diff
@@ -20,33 +20,238 @@
...
+
+Are there any examples of using Landlab I can look at?
+------------------------------------------------------
+
+The Landlab package contains a directory called ``notebook`` which contains
+Jupyter notebooks describe core concepts and give examples of using components.
+The file ``landlab/notebooks/welcome.ipynb`` provides a table of contents to
+the notebooks and is the recommended starting place. To launch an instance of
+Binder and explore the notebooks click
+`here <https://mybinder.org/v2/gh/landlab/landlab/release?filepath=notebooks/welcome.ipynb>_`.
```
Comment (nathanlyons) [inline README.rst]: Not directly pertinent to this PR: I get a 404 not found error when trying to launch binder.

File: README.rst (hunk 8 above)
Comment (nathanlyons) [reply README.rst]: Just saw this was mentioned above. Probably not needed again here.

**Comment by kbarnhart** [comment]
@nathanlyons thanks for the comments!

File: README.rst (hunk 9 above)
Comment (kbarnhart) [reply README.rst]: I sometimes get those... Not sure why. I think its because binder is temporarily down...

File: README.rst (hunk 7 above)
Comment (kbarnhart) [reply README.rst]: Yeah. These few empty links are things that will get put into the docs by #1117...

File: README.rst (hunk 5 above)
Comment (kbarnhart) [reply README.rst]: Yup. Link forthcoming with #1117

File: README.rst (hunk 4 above)
Comment (kbarnhart) [reply README.rst]: Yup. Link forthcoming with #1117

File: CONTRIBUTING.md (hunk 1 above)
Comment (kbarnhart) [reply CONTRIBUTING.md]: good point. fixed.

File: README.rst (hunk 2 above)
Comment (kbarnhart) [reply README.rst]: fixed.

File: README.rst (hunk 3 above)
Comment (kbarnhart) [reply README.rst]: fixed

File: README.rst (hunk 6 above)
Comment (kbarnhart) [reply README.rst]: fixed.

File: README.rst (hunk 9 above)
Comment (kbarnhart) [reply README.rst]: I also get a lot of "failed to connect to event stream"... but those typically just mean that I don't get to see the binder build/load log.

File: README.rst (hunk 8 above)
Comment (kbarnhart) [reply README.rst]: good point. I've deleted it.

**Comment by kbarnhart** [comment]
[Linking to the JOSS Review Issue](https://github.com/openjournals/joss-reviews/issues/1935)

**Pull request merged**